sphinx/source/docs/user_guide/quickstart.rst

```diff
@@ -7,43 +7,43 @@ Introduction
 ============
 
 Bokeh is a Python interactive visualization library that targets modern web
-browsers for presentation providing elegant, concise construction of novel
-graphics with high-performance interactivity over very large or streaming
-datasets in a quick and easy way.
+browsers for presentation. Bokeh provides quick, easy, elegant, concise 
+construction of novel graphics with high-performance interactivity over very 
+large or streaming datasets.
 
-Offering both powerful and flexible features to enable very advanced
-customizations in one hand and simplicity on the other, Bokeh exposes different
-interface levels to the users:
+To offer both simplicity and the powerful and flexible features needed for 
+advanced customizations, Bokeh exposes three interface levels to users:
 
-* a *low-level* |bokeh.models| interface that provides the most flexibility
-  to application developers.
-* an *intermediate-level* |bokeh.plotting| interface that is centered
-  around composing visual glyphs.
-* a *high-level* |bokeh.charts| interface that can be used to build complex
-  statistical plots as quickly and as simply as possible.
+* a *low-level* |bokeh.models| interface providing the most flexibility to 
+  application developers.
+* an *intermediate-level* |bokeh.plotting| interface centered around composing 
+  visual glyphs.
+* a *high-level* |bokeh.charts| interface to build complex statistical plots 
+  quickly and simply.
 
 This Quickstart focuses on the |bokeh.plotting| interface.
 
 Quick Installation
 ==================
 
-There are a few different ways to install Bokeh.
-
-If you are using the `Anaconda Python distribution`_ (which is recommended),
-enter this command at a Bash or Windows command prompt:
+There are multiple ways to install Bokeh, and we recommend the easiest one, 
+which is to use the `Anaconda Python distribution`_ and enter this command at 
+a Bash or Windows command prompt:
 
 .. code-block:: sh
 
     conda install bokeh
 
-This installs all the dependencies that you need to be ready to run Bokeh
-and we strongly recommend using it. It reduces the installation effort to
-nearly zero on any platform and configuration (including Windows). It also
-installs the examples into the ``examples/`` subdirectory of your Anaconda (or
-miniconda) installation directory.
+This installs all the dependencies that Bokeh needs. Anaconda minimizes 
+installation effort on all platforms and configurations, including Windows, 
+and also installs the examples into the ``examples/`` subdirectory of your 
+Anaconda or miniconda installation directory. (For instance, if you use 
+Miniconda with Python 3 on OS X or Linux and your Bokeh environment is called 
+bokehEnv, the examples directory would be at 
```

Comment:
I added this because I first looked in ~/miniconda3 , found nothing, and wasn't sure if the docs were incorrect, or I was misunderstanding, or what. Eventually found it, though, so we can probably assume most users will be less delayed by skepticism than I was, and will hunt it down the way I did. I'll remove the parenthetical.